.github/workflows/main.yaml

```diff
@@ -1,24 +1,21 @@
 name: unitary
 
-on: ["push", "pull_request"]
```

Comment:
not sure we want to disable test on push

Reply:
i also don't think `pull_request_target` tests the merge commit, which may be a problem. maybe we should run all three?

Reply:
I've added `push` to master and `pull_request`, but then without the integration tests